src/cmd/dev.go

```diff
@@ -286,3 +323,34 @@ func bindDevDeployFlags(v *viper.Viper) {
 
 	devDeployFlags.BoolVar(&pkgConfig.CreateOpts.NoYOLO, "no-yolo", v.GetBool(common.VDevDeployNoYolo), lang.CmdDevDeployFlagNoYolo)
 }
+
+func bindDevGenerateFlags(v *viper.Viper) {
+	generateFlags := devGenerateCmd.Flags()
+
+	generateFlags.StringVar(&pkgConfig.GenerateOpts.URL, "url", "", "URL to the source git repository")
+	generateFlags.StringVar(&pkgConfig.GenerateOpts.Version, "version", "", "The Version of the chart to use")
+	generateFlags.StringVar(&pkgConfig.GenerateOpts.GitPath, "gitPath", "", "Relative path to the chart in the git repository")
+	generateFlags.StringVar(&pkgConfig.GenerateOpts.Output, "output-directory", "./", "Output directory for the generated zarf.yaml")
+	generateFlags.StringVar(&pkgConfig.FindImagesOpts.KubeVersionOverride, "kube-version", "", lang.CmdDevFlagKubeVersion)
+}
+
+func validateDevGenerateFlags() bool {
```

Comment:
This function can be replaced by using the built in cobra MarkFlagsRequired Function. For example
`devGenerateCmd.MarkFlagRequired("url")`

Reply:
Thank you!